site/docs/v1/tech/samlv2/zendesk.adoc

```diff
@@ -37,7 +37,9 @@ Next, we'll create a lambda to take FusionAuth role information and populate the
 [source,javascript]
 ----
 function populate(samlResponse, user, registration) {
-  samlResponse.assertion.attributes.role = registration.roles;
+  if (registration && registration.role) {
+    samlResponse.assertion.attributes.role = registration.roles;
+  }
```

Comment:
`registration.role` does not exist, so this code will not be executed.  `Is this if block protecting against an expected state?

Reply:
Doh. that is supposed to be `registration.roles` in the if statement. Let me fix that.

I was seeing a situation where a user who wasn't registered for the zendesk application was going through the SAML flow. `registration` was null and the lambda was throwing an exception in the logs when accessing that null object.

Reply:
Hmm.. what version of FusionAuth are you seeing that on?

Reply:
Ah, ok. Yes you're correct, that parameter may be null. It probably shouldn't be, I'll fix that, but for now that lambda is correct.